musicdroid/src/main/java/org/catrobat/musicdroid/pocketmusic/instrument/InstrumentActivity.java

```diff
@@ -165,7 +165,18 @@ public boolean onOptionsItemSelected(MenuItem item) {
             onActionDeleteMidi();
             return true;
         } else if (id == R.id.action_play_midi) {
-            onActionPlayMidi();
+
+            //If the player is not playing then start playing and change the play icon to stop icon
+            //Else, if the player is playing then stop playing and change the stop icon to play icon
+            //The icon gets changed back when onPlayTrackComplete() is called by the midiPlayer when the entire track has been played
+
```

Comment:
As we are working like described in "Clean Code": https://www.ufm.edu/images/0/04/Clean_Code.pdf we don't need to write comments if the code is self-explanatory